x-pack/plugins/grokdebugger/public/plugin.js

```diff
@@ -12,7 +12,15 @@ import { PLUGIN } from '../common/constants';
 import { registerFeature } from './register_feature';
 
 export class GrokDebuggerUIPlugin {
+  constructor(initializerContext) {
+    this.config = initializerContext.config.get();
+  }
+
   setup(coreSetup, plugins) {
+    if (this.config?.enabled === false) {
```

Comment:
I don't _think_ this check is needed (and elsewhere in the code). IIRC the plugin will automatically be disabled by core if the `enabled` flag is set to false.

Reply:
I can test that and verify. If they are needed I will remove them. I know we have these checks in some other plugins.

Reply:
@alisonelizabeth remove these and simplified the code since core is handling this. Not sure why we manually handle this in some other plugins. Thanks for the call out.